src/components/UI/Form/FormSubmission/FormSubmission.js

```diff
@@ -0,0 +1,54 @@
+import { Container, makeStyles, Typography, Grid, Button } from '@material-ui/core';
+import React from 'react';
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    marginTop: theme.spacing(8),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  form: {
+    width: '100%',
+    marginTop: theme.spacing(3),
+    '& .MuiTextField-root': {
+      margin: theme.spacing(1),
+    },
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+    float: 'right'
+  },
+  flex: {
+    display: 'flex',
+  },
+}))
+
+const FormSubmission = (props) => {
```

Comment:
Ошибка eslint, нужно на  деструктуризацию переделать

```suggestion
const FormSubmission = (props) => {
```

Reply:
сделана деструктуризация